index.js

```diff
@@ -1,5 +1,5 @@
 import { NativeModules } from 'react-native';
 
-const { SalesforceChat } = NativeModules;
+const { RNSalesforceChat } = NativeModules;
```

Comment:
When testing this on the actual import, I noticed that `SalesforceChat` is exported here, not `RNSalesforceChat`. I had to re-release the package and bump the version time. Other than that, this PR looks good!